utils/containers/hash/hash_map_generator.bzl

```diff
@@ -48,6 +48,7 @@ def hash_map_generate(mapped_type):
             "//common/trinary:flex_trit",
             "//utils/handles:rand",
             "//utils/containers/hash:hash243_set",
+            "//consensus:model",
```

Comment:
Same thing as the include, maybe dependency should be a parameter of the rule and not embedded in the generator.